### PR TITLE
Zombies own 2021

### PR DESCRIPTION
**Pull request by Bibby0110**
# General Documentation

### Intent of your Pull Request
Makes zombies less shit. Can't die when organs fall out, ignores damage slow down to a certain degree
Ability cooldowns nerfed.

### Why is this change good for the game?

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
100 seconds to open welded door
80 to open bolted door
200 to open welded & bolted door

3 minute cooldown on adrenaline for runners
2.5 minute cooldown on tank
tier 2 time halved
Infect chance only based on brute damage now
Increase damage of being infected

### What general grouping does this PR fall under? 
Zombie balancing

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
no

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
see above

# Changelog

:cl:  
tweak: Tweaked zombie values, probably still dogshit, but less so now
/:cl:

### CODE REVIEW

File: code/modules/antagonists/zombie/abilities/tank.dm [hunk 1]
```diff
@@ -3,10 +3,10 @@
 	desc = "Gives you a moderate armor boost for a few seconds. Heals 75% of your brute and fire damage."
```
Comment (redmoogle) [inline code/modules/antagonists/zombie/abilities/tank.dm]: you changed healing from 75% to 60%

**Review by redmoogle — CHANGES_REQUESTED** [review]

File: code/game/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -1272,15 +1272,15 @@
```
Comment (redmoogle) [inline code/game/machinery/doors/airlock.dm:1272]: This can be optimized to do

if(welded && locked)

if(welded)

if(locked)

not a request, but a thought

Comment (Bibby0110) [reply code/game/machinery/doors/airlock.dm:1272]: no reason

File: code/modules/antagonists/zombie/abilities/tank.dm (hunk 1 above)
Comment (Bibby0110) [inline code/modules/antagonists/zombie/abilities/tank.dm]:
```suggestion
	desc = "Gives you a moderate armor boost for a few seconds. Heals 60% of your brute and fire damage."
```

**Review by JamieD1 — APPROVED** [review]
